packages/commonwealth/server/controllers/server_admin_methods/get_stats.ts

```diff
@@ -0,0 +1,120 @@
+import { Op, QueryTypes, WhereOptions } from 'sequelize';
+import { AppError } from '../../../../common-common/src/errors';
+import { CommunityAttributes } from '../../models/community';
+import { UserInstance } from '../../models/user';
+import { ServerAdminController } from '../server_admin_controller';
+
+export const Errors = {
+  NotAdmin: 'Must be a site admin',
+  CommunityNotFound: 'Community not found',
+};
+
+export type GetStatsOptions = {
+  user: UserInstance;
+  communityId?: string;
+};
+
+export type GetStatsResult = {
+  lastMonthNewCommunities: Array<string>;
+  totalStats: {
+    numCommentsLastMonth: number;
+    numThreadsLastMonth: number;
+    numPollsLastMonth: number;
+    numReactionsLastMonth: number;
+    numProposalVotesLastMonth: number;
+    numMembersLastMonth: number;
+    numGroupsLastMonth: number;
+  };
+};
+
+export async function __getStats(
+  this: ServerAdminController,
+  { user, communityId }: GetStatsOptions,
+): Promise<GetStatsResult> {
+  if (!user.isAdmin) {
+    throw new AppError(Errors.NotAdmin);
+  }
+
+  // community is optional
+  let community: CommunityAttributes | undefined = undefined;
+  if (communityId) {
+    community = await this.models.Community.findByPk(communityId);
+    if (!community) {
+      throw new AppError(Errors.CommunityNotFound);
+    }
+  }
+
+  const oneMonthAgo = new Date();
+  oneMonthAgo.setDate(oneMonthAgo.getDate() - 30);
+
+  const where = {
+    created_at: {
+      [Op.gte]: oneMonthAgo,
+    },
+  };
+  const whereChain: WhereOptions<{
+    created_at: any;
+    chain: string;
+  }> = {
+    ...where,
+  };
+  const whereCommunityId: WhereOptions<{
+    created_at: any;
+    community_id: string;
+  }> = { ...where };
+
+  if (community) {
+    whereChain.chain = communityId;
+    whereCommunityId.community_id = communityId;
+  }
+
+  const [
+    lastMonthNewCommunities,
+    numCommentsLastMonth,
+    numThreadsLastMonth,
+    numReactionsLastMonth,
+    numProposalVotesLastMonth,
+    numPollsLastMonth,
+    numMembersLastMonth,
+    numGroupsLastMonth,
+  ] = await Promise.all([
```

Comment:
Probably not as efficient as a single sql query, but good enough. Approving

Reply:
Hmm interesting. This query you suggest above does seem to only generate one query plan with the append.

I was thinking more along the lines of:
```
explain analyze SELECT
  (SELECT COUNT(*) FROM "Comments"),
  (SELECT COUNT(*) FROM "Threads"),
  (SELECT COUNT(*) FROM "Communities"),
  (SELECT COUNT(*) FROM "Groups");
```

Which seems to produce multiple query plans instead of using append (Although maybe not run in parallel, depends on how the sql planner decides to run them).